pygerduty/__init__.py

```diff
@@ -460,6 +460,7 @@ class Service(Container):
     def __init__(self, *args, **kwargs):
         Container.__init__(self, *args, **kwargs)
         self.email_filters = EmailFilters(self.pagerduty, self)
+        self.escalation_policy = EscalationPolicies(self.pagerduty, self)
```

Comment:
Can you make this `self.escalation_policies` to have consistent pluralization with other Container instances.